src/apikey/lib/controllers/ping.js

```diff
@@ -1,5 +1,13 @@
 const { healthTimeout } = require('config');
 
+/**
+ * Executes a promise but cuts it off after a while if it has not been resolved.
+ * this function is used for healthcheck routes.
+ *
+ * @param {Promise} p1
```

Comment:
Missing doc for params
```suggestion
 * @param {Promise} p1 Promise to be executed which will be
 * stopped if it does not solve after a certain time.
```